### DISCUSSION TITLE
custom buttons in forms

### DISCUSSION THREAD

**Discussion by photojo** [filamentphp/filament#5646] [Help] [answered]
I'm using 
getTableHeaderActions() 
to get buttons in header, f.e. config-modal or data-export. It works fine!

But how can I get one of these buttons in a form, instead of in the header.

For example: in my config-modal there should be a button, which calls a certain function.

->form([
                    Toggle::make('ap_statistik_schoolview')
                        ->label('Schulansicht freischalten')
                        ->hint('Wenn die Schulansicht abgeschaltet ist, so wird auch bei passendem Datum keine Statistik angezeigt.'),
                      BUTTON with ACTION ???
])

**Comment by Z3d0X** [comment, marked as answer]
The "Drop-In Action" plugin might  what u are looking for: https://filamentphp.com/plugins/drop-in-action

**Comment by photojo** [comment]
Thank you! That works!
